src/components/notifications/Cellsynt.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <div class="mb-3">
+        <label for="cellsynt-login" class="form-label">{{ $t("Username") }}</label>
+        <input id="cellsynt-login" v-model="$parent.notification.cellsyntLogin" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="cellsynt-key" class="form-label">{{ $t("Password") }}</label>
+        <HiddenInput id="cellsynt-key" v-model="$parent.notification.cellsyntPassword" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="cellsynt-Originatortype" class="form-label">{{ $t("Originator type") }}</label>
+        <select id="cellsynt-Originatortype" v-model="$parent.notification.cellsyntOriginatortype" :required="true" class="form-select">
+            <option value="alpha">{{ $t("Alpha (recommended)") }}</option>
+            <option value="numeric">{{ $t("Numeric") }}</option>
```

Comment:
I think here `Telephone number` might be simpler to understand. What do you think?

Reply:
Im not sure. But for my projects it doesnt matter. 
Maybe lock for alpha and text "uptime-kuma"? Then user only have to input destination?

Reply:
I think changing this to `Telephone number` and the `Alpha` to `Alphanumeric` is fine